README.md

```diff
@@ -26,64 +29,64 @@ Basic Concept
 Getting Started
 ---------------
 
-- You need Python3. We tested primarily with **python 3.5**
-- This was tested with a recent Ubuntu based Linux.
+- You need Python3.6 and above for installing tanner.
+- This was tested with a recent Ubuntu-based Linux.
 
+### Steps to install TANNER
 
-### Setup Redis
-
+#### Step 1: Setup Redis
 
 1. Install the Redis: ``sudo apt-get install redis-server``
-2. Start it on ``localhost`` with default ``port``: ``redis-server``
-
-### Setup PHP Sandbox
+2. Run ``redis-server`` (to start it on `localhost` with default `port`)
 
+#### Step 2: Setup PHP Sandbox
 
-1. For PHP Sandbox setup, see sandbox [manual](https://github.com/mushorg/phpox)
-2. In PHP Sandbox directory, run sandbox: ``sudo python3 sandbox.py``
+1. Clone the repo using:  ``git clone https://github.com/mushorg/phpox.git`` (preferably in your home directory).
+2. Go to PHP Sandbox directory: ``cd phpox``
+2. Run sandbox: ``sudo python3 sandbox.py``
 
-### Setup Docker
-
-
-1. Install [docker](https://docs.docker.com/engine/installation/linux/ubuntu/)
-2. Pull the required image to use [default : ``busybox:latest``]
-
-### Setup and run TANNER
+For more help on PHP Sandbox setup, see sandbox [manual](https://github.com/mushorg/phpox)
 
+#### Step 3: Setup and run TANNER
 
 1. Get TANNER: `git clone https://github.com/mushorg/tanner.git`
 2. Go to the TANNER source  directory: ``cd tanner``
 3. Install requirements: `pip3 install -r requirements.txt`
 4. Install TANNER: ``sudo python3 setup.py install``
 5. Run TANNER: ``sudo tanner``
+6. (Optional) For runnning TANNER Api ``sudo tannerapi``
+7. (Optional) For runnning TANNER Web ``sudo tannerweb``
 
-### Run Tanner Api
+(Recommended) You should to bind to 0.0.0.0 when running in
+<i>production</i> and on a different host than SNARE.
 
-Run ``sudo tannerapi``
+### Install and run TANNER using docker container
 
-### Run Tanner WebUI
+  In case you want to run the TANNER service using docker or facing any problem
+  in setting up TANNER on your machine, you can follow these steps.
 
-Run ``sudo tannerweb``
+#### Setup Docker
```

Comment:
This step should be before tanner setup, since tanner using docker inside. It will not work without docker

Reply:
I have made the required changes